src/Control/Parallel/Class.purs

```diff
@@ -0,0 +1,34 @@
+module Control.Monad.Parallel.Class 
+  ( MonadPar, par
+  , MonadRace, race
+  , Parallel(), inParallel, runParallel
+  ) where
+
+import Prelude
+import Control.Alt
+
+class (Monad m) <= MonadPar m where
+  par :: forall a b. m (a -> b) -> m a -> m b
```

Comment:
This is purely aesthetic, but since this is not called apply or ap or anything, and because it's meant to suggest independent production of two values, I prefer the alternative applicative encoding:

```purescript
  par :: forall a b. m a -> m b -> m (Tuple a b)
```

The docs should clearly state that unlike `ap` & friends, the order of effects is not guaranteed to be left-to-right.